schemas/dependencies/jenkins-instance-1.yml

```diff
@@ -50,6 +50,39 @@ properties:
       required:
       - name
       - keep_hours
+  workerFleets:
+    type: array
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        account:
+          "$ref": "/aws/tenant_accounts-1.yml#/properties/account"
+        identifier:
```

Comment:
This identifier needs to match an asg identifier, correct? If so, we should document it here.